bittensor/_dendrite/__init__.py

```diff
@@ -190,7 +191,7 @@ def manager_serve(cls, config, wallet, receptor_pool = None):
                 max_worker_threads = config.dendrite.max_worker_threads,
                 max_active_receptors = config.dendrite.max_active_receptors
             )
-        ManagerServer.register('get_receptorpool', callable=lambda:receptor_pool,exposed=['forward','backward','get_receptors_state'])
-        manager = ManagerServer(address=('', 50000), authkey=b'abracadabra')
+        ManagerServer.register('get_receptorpool', callable=lambda:receptor_pool,exposed=['forward','backward','get_receptors_state', 'get_total_requests'])
+        manager = ManagerServer(address=('', 4098), authkey=b'abracadabr')
```

Comment:
For security reasons, can we change the authkey to the wallet hotkey?

Reply:
Goooddd idea! changed!